projects/services-map-state/src/lib/map-state.service.ts

```diff
@@ -59,6 +59,28 @@ export class MapStateService {
     this.setMapState(state);
   }
 
+  public setViewAngle(angle: number, notifier: IMapState['options']['notifier'] = 'user') {
+    if (!angle) {
```

Comment:
In here, I would allow to pass 0 as an angle. Otherwise, users must use small values to reset the ViewAngle.